docs/writing-policies/go/07-automate.md

```diff
@@ -65,11 +63,11 @@ jobs:
       OCI_TARGET: ghcr.io/kubewarden/policies/safe-labels
 ```
 
-Pushing a tag named `v0.1.0` will lead to the creation and publishing of the
+Pushing a tag named `v0.1.0` leads to the creation and publishing of the
 OCI artifact called `ghcr.io/kubewarden/policies/safe-labels:v0.1.0`.
 
-A GitHub Release named `Release v0.1.0` will be created. The release will
-include the following assets:
+It creates a GitHub release named `Release v0.1.0`.
+The release includes the following assets:
 
-* Source code compressed as `zip` and `tar.gz`
-* A file named `policy.wasm` that is the actual WebAssembly policy
+- Source code compressed as `zip` and `tar.gz`
+- A file named `policy.wasm`; this is the actual WebAssembly policy
```

Comment:
I think we need to rework the example section. This is using the old approch. Using the current reusable workflows, we should need to change the input for the `kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml` workflow. Take a look on this example from the `safe-labels` policy:

```yaml 
on:
  push:
    branches:
    - main
    tags:
    - 'v*'

name: Release policy

jobs:

  test:
    name: run tests and linters
    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-go.yml@v3.1.11

  release:
    needs: test
    permissions:
      # Required to create GH releases
      contents: write
      # Required to push to GHCR
      packages: write
      # Required by cosign keyless signing
      id-token: write

    uses: kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml@v3.1.11
    with:
      oci-target: ghcr.io/${{ github.repository_owner }}/policies/safe-labels #note here. This is line that requires a  change
```
This yaml came from [here](https://github.com/kubewarden/safe-labels-policy/blob/main/.github/workflows/release.yml). I think we can just copy it. Or copy only the call of the `kubewarden/github-actions/.github/workflows/reusable-release-policy-go.yml` workflow

Reply:
So which? Further suggestions?

Reply:
I think I prefer the current approach that only pushes the policy to an OCI registry. Using the reusable workflow is opaque to newcomers, and we do things there that aren't actually explained (e.g: signing, artifacthub) and are explained in ./distributing-policies.md.

@jvanz wdyt?

Reply:
Fair enough. I agree with @viccuad. Let's keep the current approach.